packages/ui/src/lib/Button/index.test.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { BrowserRouter as Router } from 'react-router-dom';
+import { render, screen, fireEvent } from '@testing-library/react';
+// import { Button, IconButton, LinkButton } from './index';
+import { Button, LinkButton } from './index';
+import { BrowserRouter } from 'react-router-dom';
```

Comment:
dont see how its used, also another import above

Reply:
not sure myself.